R/install-remote.R

```diff
@@ -108,3 +116,46 @@ remote_download <- function(x, quiet = FALSE) UseMethod("remote_download")
 remote_metadata <- function(x, bundle = NULL, source = NULL) UseMethod("remote_metadata")
 remote_package_name <- function(remote, ...) UseMethod("remote_package_name")
 remote_sha <- function(remote, ...) UseMethod("remote_sha")
+
+package2remote <- function(x, ...) {
+  x <- packageDescription(x)
+  if (is.null(x$RemoteType)) {
+    return(NULL)
```

Comment:
What do you think about having a CRAN package type here? That might make it easy for `update_packages()` to work with everything